logging-modules/log4j2-programmatic-configuration/modify-xml-configuration/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration monitorInterval="60">
+    <Appenders>
+        <Console name="console" target="SYSTEM_OUT">
+            <PatternLayout pattern="[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+    	<Logger name="com" additivity="false" level="info">
```

Comment:
Let's make the indentation consistent. Seems that in some places in this file, it is 2 spaces while in others it is 4.